docs/book/src/tasks/upgrading-clusters.md

```diff
@@ -47,6 +47,22 @@ that if a specific machine image is specified, it has to match the Kubernetes ve
 `KubeadmControlPlane` spec. In order to only trigger a single upgrade, the new `MachineTemplate` should be created first
 and then both the `Version` and `InfrastructureTemplate` should be modified in a single transaction.
 
+#### How to trigger a rollout irrespective of spec changes
+
+A `KubeadmControlPlane` resource has a field `UpgradeAfter` that can be set to a timestamp
+(RFC-3339) after which a rollout should be triggered regardless of whether there were any changes
+to the `KubeadmControlPlane.Spec` or not. This would roll out replacement control plane nodes
+which can be useful e.g. to perform certificate rotation, reflect changes to machine templates,
+move to new machines, etc.
+
+To do the same for machines managed by a `MachineDeployment` it's enough to make an arbitrary
```

Comment:
It might be good to also call out the `clusterctl alpha rollout restart` command here: https://github.com/kubernetes-sigs/cluster-api/blob/master/cmd/clusterctl/cmd/rollout/restart.go

Reply:
Ah, nice, didn't know about that. This can probably replace the `kubectl patch` approach, right?

Reply:
I believe so, yes

Reply:
OK I got this change prepared but I just realised `clusterctl alpha rollout` is only in master and not in the latest released version. Docs are not versioned, so just wanted to check what's the policy, is it OK to reference unreleased features in documentation?

Reply:
FYI doc are versioned
release-0-3 branch goes to https://cluster-api.sigs.k8s.io
main branch goes to https://master.cluster-api.sigs.k8s.io

Reply:
Done, thanks!